src/hotspot/share/runtime/objectMonitor.cpp

```diff
@@ -987,6 +987,13 @@ void ObjectMonitor::ReenterI(JavaThread* current, ObjectWaiter* currentNode) {
     guarantee(v == ObjectWaiter::TS_ENTER || v == ObjectWaiter::TS_CXQ, "invariant");
     assert(owner_raw() != current, "invariant");
 
+    // This thread has been notified so reacquire the lock.
```

Comment:
nit typo: s/so reacquire/so try to reacquire/

Reply:
Fixed.